aea/test_tools/test_contract.py

```diff
@@ -143,20 +143,24 @@ def refill_from_faucet(
             raise ValueError("Balance not increased!")  # pragma: nocover
 
     @staticmethod
-    def sign_send_confirm_receipt_transaction(
-        tx: JSONLike, ledger_api: LedgerApi, crypto: Crypto, sleep_time: float = 1.0
+    def sign_send_confirm_receipt_multisig_transaction(
```

Comment:
Let's write a code comment (not docstring) for this function, that this function and `sign_send_confirm_receipt_transaction` below are there for backwards compatibility purposes. Next major release we can combine them (remove the one below and use this one.)

Reply:
Let's write a code comment (not docstring) for this function with the following format, explaining the situation so for the next major release we know what parts we can clean up.

`# BACKWARDS COMPATIBILITY: ....`